### PR TITLE
CTCP-132 : remove date goods unloaded njk file and create a separate …

### PR DESCRIPTION
**Pull request by kenneth-rayner**
…dateGoodsUnloadedRejectionView in Twirl

### CODE REVIEW

File: test/views/DateGoodsUnloadedRejectionViewSpec.scala [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import base.{AppWithDefaultMockFixtures, SpecBase}
+import forms.DateGoodsUnloadedFormProvider
+import play.api.data.Form
+import play.twirl.api.HtmlFormat
+import views.behaviours.DateInputViewBehaviour
+import views.html.DateGoodsUnloadedRejectionView
+
+import java.time.{Clock, LocalDate}
+
+class DateGoodsUnloadedRejectionViewSpec extends SpecBase with DateInputViewBehaviour with AppWithDefaultMockFixtures {
```
Comment (jackheslop96) [inline test/views/DateGoodsUnloadedRejectionViewSpec.scala:28]: Should just need to extend `DateInputViewBehaviour` here I believe

Comment (kenneth-rayner) [reply test/views/DateGoodsUnloadedRejectionViewSpec.scala:28]: Done

**Review by jackheslop96 — APPROVED** [review]

File: app/views/DateGoodsUnloadedRejectionView.scala.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+@this(
+        main_template: MainTemplate,
+        errorSummary: DateErrorSummary,
+        formHelper: FormWithCSRF,
+        inputDate: InputDate,
+        button: Button,
+        caption: HeadingCaption
+)
+
+@(mrn: String, arrivalId: ArrivalId, form: Form[LocalDate])(implicit request: Request[_], messages: Messages)
```
Comment (kevincoatsworth) [inline app/views/DateGoodsUnloadedRejectionView.scala.html:30]: Is the MRN needed in this view? The arrivalId doesn't appear to be used

Comment (kenneth-rayner) [reply app/views/DateGoodsUnloadedRejectionView.scala.html:30]: It's used in  the @caption(mrn) view line 41
The arrivalId is used in the onSubmit(arrivalId) line 37

Comment (kevincoatsworth) [reply app/views/DateGoodsUnloadedRejectionView.scala.html:30]: The previous view doesn't seem to pass the mrn into the view so I was presuming the arrivalId would be the caption, though I may be mistaken. I meant the arrivalId wasn't being used in the caption, I wasn't very clear!

Comment (kenneth-rayner) [reply app/views/DateGoodsUnloadedRejectionView.scala.html:30]: Mrn appears to have been missed off the njk version

**Review by kevincoatsworth — APPROVED** [review]

**Pull request merged**